src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inverseRootArray = Object.entries(robot)
+    .map(([key, value]) => [value, key]);
+
+  const result = Object.fromEntries(inverseRootArray);
```

Comment:
Object.entries, map, Object.fromEnties - you use 3 iterations instead of use one cycle
Try to optimize your code